aodndata/aatams/aatams_sattag_nrt.py

```diff
@@ -75,37 +72,36 @@ def preprocess(self):
-            ]
-            logger.info(
-                "NRT update requested: Comparing timestamps in the metadata files %s and %s"
-                % (old_metadata_file, new_metadata_file),
-            )
-            self.broker.download(old_metadata_file, self.temp_dir)
-            if self.is_update_required(old_metadata_file.local_path):
-                for remote_file in self.previous_files:
-                    filename = remote_file.name
-                    file_to_remove = PipelineFile(
-                        filename, is_deletion=True, dest_path=dest_path(filename),
-                    )
-                    file_to_remove.publish_type = (
-                        PipelineFilePublishType.DELETE_UNHARVEST
-                    )
-                    self.file_collection.add(file_to_remove)
-            else:
-                raise InvalidFileContentError(
-                    "File %s within %s contains older entries than current NRT state from %s"
-                    % (
-                        new_metadata_file.name,
-                        self.input_file_object.name,
-                        old_metadata_file.name,
-                    ),
-                )
+    # TODO enable below and test for removal of old NRT files
+    # def process(self):
+    #     """Process NRT files, removing the previous files if content of new file is
+    #     updated."""
+
+    #     self.previous_files = self.state_query.query_storage(dest_path(""))
+    #     if self.previous_files:
+    #         old_metadata_file = self.previous_files[metadata_index(self.previous_files)]
+    #         new_metadata_file = self.file_collection[
+    #             metadata_index(self.file_collection)
+    #         ]
+    #         logger.info("NRT update requested: Comparing timestamps in the metadata files %s and %s", old_metadata_file, new_metadata_file)
+    #         remote_collection = RemotePipelineFileCollection(old_metadata_file)
+    #         self.state_query.download(remote_collection)
+    #         if self.is_update_required(old_metadata_file.local_path):
+    #             for remote_file in self.previous_files:
+    #                 filename = remote_file.name
+    #                 file_to_remove = PipelineFile(
+    #                     filename, is_deletion=True, dest_path=dest_path(filename),
+    #                 )
+    #                 file_to_remove.publish_type = (
+    #                     PipelineFilePublishType.DELETE_UNHARVEST
+    #                 )
+    #                 self.file_collection.add(file_to_remove)
+    #         else:
+    #             raise InvalidFileContentError(
+    #                 "File %s within %s contains older entries than current NRT state from %s"
+    #                 % (
+    #                     new_metadata_file.name,
+    #                     self.input_file_object.name,
+    #                     old_metadata_file.name,
+    #                 ),
+    #             )
```

Comment:
why not just removing it ? Do you think this will be re-used in the future

Reply:
@evacougnon mentioned that it could be turned on in the future, so I thought it would be better to keep the work done here than to throw it away.